py/server/tests/test_table_factory.py

```diff
@@ -52,6 +52,10 @@ def test_time_table(self):
         self.assertTrue(t.is_refreshing)
         self.assertEqual("2021-11-06T13:21:00.000000000 NY", t.j_table.getColumnSource("Timestamp").get(0).toString())
 
+        t = time_table(1000000000)
+        self.assertEqual(1, len(t.columns))
+        self.assertTrue(t.is_refreshing)
```

Comment:
Make sure you test all 4 branches based on type.  The (int,str) branch is not tested.